pkg/processor/processor.go

```diff
@@ -247,11 +247,6 @@ func NewReportRunner(cfg *config.Config, dbConn *gorm.DB, sf common.SalesforceCl
 		return nil, err
 	}
```

Comment:
Do you still need any of the above code?

Reply:
Yes, that's the working directory of the shell script for the processor.

Reply:
Ack, just seemed like the new code was doing the same thing.

Reply:
The `basepath` corresponds to the mount point in `docker-compose` where the shared folder is mounted on both containers. That needs to be set in both configuration files (which is a little hackish at the moment and needs some cleaning). The processor then creates a directory underneath that where it unpacks the sos reports and runs hotsos. That step is as it was before.